crates/bevy_scene/src/scene_spawner.rs

```diff
@@ -37,6 +38,7 @@ impl InstanceId {
     }
 }
 
+/// Handles spawning and despawning scenes in the world, either synchronously or batched through the `scene_spawner_system`.
```

Comment:
Could this include links to the synchronous functions and non-synchrnous function as point list?